src/components/HomepageFeatures.js

```diff
@@ -181,7 +181,7 @@ export default function HomepageFeatures() {
                                             <a href="https://rancher-users.slack.com/archives/C01GKHKAG0K" className="font-weight-bolder rfs-10 text-white">Join our Slack Community</a>
                                         </p>
                                         <p className="mb-2">
-                                            <a href="https://youtu.be/EKDtheJxQN4" className="font-weight-bolder rfs-10 text-white">Watch the latest meetup</a>
+                                            <a href="https://youtu.be/ZFtOZXj_PZ8" className="font-weight-bolder rfs-10 text-white">Watch the latest meetup</a>
```

Comment:
let's use the latest meetup video with `https://www.youtube.com/watch?v=uhdqD7_Mwzw`, thanks.

Reply:
Changed the link for Blog.